dotnet/src/InternalUtilities/samples/InternalUtilities/TestConfiguration.cs

```diff
@@ -72,6 +73,13 @@ public class OpenAIConfig
         public string ApiKey { get; set; }
     }
 
+    public class AzureAIInferenceConfig
+    {
+        public string ServiceId { get; set; }
+        public string Endpoint { get; set; }
+        public string? ApiKey { get; set; }
+    }
```

Comment:
This configuration should be added to `testsettings.json` and `dotnet-build-and-test.yml`. Otherwise, there will be an error during merge that it can't find `AzureAIInference` configuration. I would recommend running integration tests against feature branch before merging to `main` to verify that tests are passed in CI pipeline. This exercise should save some time during actual merge to `main` branch.